impl/client/src/doc/javascript/template/vizApi/tmpl/type.tmpl

```diff
@@ -1,7 +1,207 @@
 <?js
-    var data = obj;
-    var self = this;
-    data.forEach(function(name, i) { ?>
-<span class="param-type"><?js= self.linkto(name, self.htmlsafe(name)) ?></span>
-<?js if (i < data.length-1) { ?>|<?js } ?>
-<?js }); ?>
\ No newline at end of file
+  var data = obj;
```

Comment:
This move was done in the CDF template (or the templates weren't in sync like they should), I wasn't aware that types were removed. I'm going to check this.